js/client-rooms.js

```diff
@@ -24,6 +24,12 @@
 			app.send('/cmd rooms');
 			app.user.on('change:named', this.updateUser, this);
 			this.update();
+			this.chatroomInterval = setInterval(function () {
+				if (app.curSideRoom && app.curSideRoom.id === 'rooms') {
+					app.send('/cmd rooms');
+					this.update();
```

Comment:
Was this tested? This is a crash and also seems unnecessary because `/cmd rooms`'s response already triggers an update, and until it gets a response there's nothing to update.